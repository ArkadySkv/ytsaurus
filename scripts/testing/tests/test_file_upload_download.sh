#!/bin/bash
#% NUM_MASTERS = 3
#% NUM_HOLDERS = 5

rm -f downloaded_file.txt

dd if=/dev/urandom of=some_file.txt bs=1000 count=1000 2>/dev/null

cat some_file.txt |yt upload /file

yt get /file@size
yt get /sys/chunks@сount

yt download /file > downloaded_file.txt

diff= `diff some_file.txt downloaded_file.txt`
echo "Diff: $diff"

yt remove /file
yt get /sys/chunks@count