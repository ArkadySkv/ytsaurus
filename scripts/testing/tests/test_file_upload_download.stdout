"0-1-190-6a582ced"
1000000
1
Diff: 
0
